### PR TITLE
@orta => Show price in artwork grid views

### PR DESCRIPTION
**Pull request by jorystiefel**
* Adds third line to artwork grid views showing the price when applicable
* Sizing logic was spread between the Cell and View subclasses, moved all sizing logic to Cell class since the cell owns the view (much simpler).

ios simulator screen shot sep 1 2015 3 46 54 pm

### CODE REVIEW

**Comment by jorystiefel** [comment]
Closes #750

File: Artsy/Views/Utilities/ARArtworkThumbnailMetadataView.m [hunk 1]
```diff
@@ -58,24 +57,41 @@ - (void)layoutSubviews
...
-- (void)configureWithArtwork:(Artwork *)artwork
+- (void)configureWithArtwork:(Artwork *)artwork showPriceLabel:(BOOL)showPrice
 {
     self.primaryLabel.text = artwork.artist.name;
     [self.secondaryLabel setTitle:artwork.title date:artwork.date];
+
+    if (showPrice) {
+        self.priceLabel.text = artwork.price;
+    }
+    _showPrice = showPrice;
```
Comment (alloy) [inline Artsy/Views/Utilities/ARArtworkThumbnailMetadataView.m]: Please use accessors where possible.

**Comment by jorystiefel** [comment]
@alloy updated to use accessor

**Comment by alloy** [comment]
This looks good to me :+1:  but needs a rebase first.

**Comment by alloy** [comment]
Ace!

**Pull request merged**